spec/requests/api/v1/npq/previous_funding/show_spec.rb

```diff
@@ -61,6 +61,20 @@ def stub_funding_eligibility_response(trn: nil, get_an_identity_id: nil, npq_cou
           expect(parsed_response).to eq(stubbed_response)
         end
 
+        context "with get_an_identity_id" do
```

Comment:
`..get_an_identity_id and trn`

Reply:
This is nested inside "with trn". So the contexts build up the full test name into:
`GET /api/v1/npq/previous_funding when authorized with trn with get_an_identity_id returns correct response`